bin/update-ips.bash

```diff
@@ -223,7 +223,7 @@ get_swift_url() {
   fi
 
   os_token=$(echo "${response}" | grep -oP "x-subject-token:\s+\K\S+")
-  swift_url=$(echo "${response}" | tail -n +15 | jq -r '.[].catalog[] | select( .type == "object-store" and .name == "swift") | .endpoints[] | select(.interface == "public" and .region == "'"${swift_region}"'") | .url')
+  swift_url=$(echo "${response}" | tail -n +14 | jq -r '.[].catalog[] | select( .type == "object-store" and .name == "swift") | .endpoints[] | select(.interface == "public" and .region == "'"${swift_region}"'") | .url')
```

Comment:
Is there any way to make this not rely on specific line numbers?

Reply:
I can change it to calls to the api, then i can skip the complete tail part. But I can test around a bit more